swaggerui/swagger.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
Is this file auto-generated? It has it generated inside the build container, but also is here.

If it is auto-generated, why do we need it? Unless we are assuming that we want it checked in for running locally?

Reply:
yes. removed